### PR TITLE
DEVDOCS-4339: [maintenance] Price Lists, update MAP definition

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4339](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4339)

## What changed?
• Update MAP definition
• Clean up grammar issues

### CODE REVIEW

**Comment by slsriehl** [comment]
@bc-tgomez she lints! take a look to see whether your changes are as intended.

File: reference/price_lists.v3.yml [hunk 1]
```diff
@@ -2631,220 +2630,373 @@ paths:
...
+        Filter items by date_created.
+      required: false
+      in: query
+      schema:
+        type: string
+        format: date-time
+    FilterIncludePriceRecordParam:
+      name: include
+      description: |
+        Sub-resources to include on a price record, in a comma-separated list. Valid expansions currently include `bulk_pricing_tiers` and `sku`.  Other valies will be ignored.
```
Comment (bc-traciporter) [inline reference/price_lists.v3.yml:2765]:
```suggestion
        Sub-resources to include on a price record, in a comma-separated list. Valid expansions currently include `bulk_pricing_tiers` and `sku`.  Other values will be ignored.
```

**Review by bc-traciporter — COMMENTED** [review]
I found one typo but looks good otherwise.

**Review by stanzikratelbc — APPROVED** [review]
Made some minor updates for stylistic changes. Looks good to me now. Ready to merge.

**Pull request merged**